### PR TITLE
Add local ephemeral storage alpha feature in managing compute resource

### PR DESCRIPTION
**Pull request by jingxu97**
Since 1.8, we add the local ephemeral storage alpha feature as one
resource type to manage. Add this feature into the doc.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5522)

### CODE REVIEW

**Comment by zacharysarah** [comment]
@jingxu97 :wave: Please change the base branch to release-1.8.

**Comment by jingxu97** [comment]
@zacharysarah What is the good way to change the base branch? Thanks!

**Comment by zacharysarah** [comment]
@jingxu97 https://github.com/blog/2224-change-the-base-branch-of-a-pull-request

**Comment by jingxu97** [comment]
@zacharysarah thanks! I changed to 1.8 release.

**Comment by zacharysarah** [comment]
Docs LGTM.

File: docs/concepts/configuration/manage-compute-resources-container.md [hunk 1]
```diff
@@ -303,7 +303,55 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
...
+Each Container of a Pod can specify one or more of the following:
+
+* `spec.containers[].resources.limits.ephemeral-storage`
+* `spec.containers[].resources.requests.ephemeral-storage`
+
+Limits and requests for `ephemeral-storage` are measured in bytes. You can express storage as
+a plain integer or as a fixed-point integer using one of these suffixes:
+E, P, T, G, M, K. You can also use the power-of-two equivalents: Ei, Pi, Ti, Gi,
+Mi, Ki. For example, the following represent roughly the same value:
+
```
Comment (saad-ali) [inline docs/concepts/configuration/manage-compute-resources-container.md]: Please add a section about what `ephemeral-storage` is and how it is measured. For example, it includes overlayFS, etc.

**Comment by steveperry-53** [comment]
@zacharysarah OK to give this Docs LGTM?

**Comment by zacharysarah** [comment]
@jingxu97 :wave: Thanks for adding the additional material. I made a few edits for clarity; LGTM! :shipit: 

/cc @steveperry-53 :wave:

**Pull request merged**